docs/admin/access.rst

```diff
@@ -1,333 +1,442 @@
+.. _custom-acl:
 
-.. note::
+Custom access control
+---------------------
 
-   Only the first rule will be used. So if you set all of
-   :guilabel:`Component list`, :guilabel:`Components` and :guilabel:`Project`,
-   only :guilabel:`Component list` will be applied.
+.. include:: /snippets/not-hosted-libre.rst
 
-An additional step is performed if checking permission for the translation:
+The permission system is based on groups and roles, where roles define a set of
+permissions, and groups link them to users and translations, see
+:ref:`auth-model` for more details.
 
+The most powerful features of the Weblate's access control system for now are
+available only through the :ref:`Django admin interface <admin-interface>`. You
+can use it to manage permissions of any project. You don’t necessarily have to
+switch it to `Custom` :ref:`access control <acl>` to utilize it. However
+you must have superuser privileges in order to use it.
 
-4. The group :guilabel:`Languages` is matched against accessed translations, it is ignored for component- or project-level access.
+If you are not interested in details of implementation, and just want to create a
+simple-enough configuration based on the defaults, or don't have a site-wide access
+to the whole Weblate installation (like on `Hosted Weblate <https://hosted.weblate.org/>`_),
+please refer to the :ref:`access-simple` section.
 
-.. hint::
+Common setups
++++++++++++++
 
-   Use :guilabel:`Language selection` or :guilabel:`Project selection`
-   to automate inclusion of all languages or projects.
+This section contains an overview of some common configurations you may be
+interested in.
 
-Checking access to a project
-++++++++++++++++++++++++++++
+Site-wide permission management
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-A user has to be a member of a group linked to the project, or any component
-inside that project. Having membership is enough, no specific permissions are needed to
-access the project (this is used in the default `Viewers` group, see
-:ref:`default-groups`).
+To manage permissions for a whole instance at once, add users to
+appropriate :ref:`default groups <default-groups>`:
 
-Checking access to a component
-++++++++++++++++++++++++++++++
+* `Users` (this is done by default by the
+  :ref:`automatic group assignment <autogroup>`).
+* `Reviewers` (if you are using :ref:`review workflow <reviews>` with dedicated
+  reviewers).
+* `Managers` (if you want to delegate most of the management operations to somebody
+  else).
 
-A user can access the unrestricted component once able to access the containing
-project. With :ref:`component-restricted` turned on, access to the component
-requires explicit permission to that component (or a component list it is in).
+You should keep all projects configured as `Public` (see :ref:`acl`), otherwise
+the site-wide permissions provided by membership in `Users` and `Reviewers` groups
```

Comment:
```suggestion
the site-wide permissions provided by membership in the `Users`, and `Reviewers` groups
```

Reply:
the comma shouldn’t be there so I remove it and committed